json_exception_handler.py

```diff
@@ -0,0 +1,36 @@
+from flask import jsonify
+from werkzeug import exceptions
+
+
+class JsonExceptionHandler(object):
+    def __init__(self, app=None):
+        if app:
+            self.init_app(app)
+
+    def json_dict(self, status_code, message):
```

Comment:
What does `json_dict` do if the status code is not one of the codes listed? Should we allow that to pass through?